packages/@icon-magic/build/test/fixtures/nav-icons/home/tmp/filled-1.svg

```diff
@@ -0,0 +1 @@
+p1
```

Comment:
Did you mean to check in these files in the `tmp` directory?

Reply:
Test results are writing this and I've updated the output path not be /tmp